src/main/resources/db/changelog/ddl/0010_create_temporary_certificate_tables.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <changeSet author="nathan.russell@valtech.com" id="0010-EIP1-3610-create-temporary-certificate-table" context="ddl">
+        <createTable tableName="temporary_certificate">
+            <column name="id" type="uuid">
+                <constraints nullable="false" primaryKey="true" unique="true"
+                             primaryKeyName="temporary_certificate_pkey"/>
+            </column>
+            <column name="certificate_number" type="varchar(20)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="gss_code" type="varchar(80)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="source_type" type="varchar(20)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="source_reference" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="application_reference" type="varchar(255)"/>
+            <column name="certificate_template_version" type="varchar(20)">
```

Comment:
This is the equivalent of `vac_version` in `print_request` which for a main cert is currently hardcoded as "A" (it means more to the print provider than us!)

For temporary certs, where we are generating the cert from a PDF template, perhaps its useful to have a template version reference?

Reply:
Maybe `template-doc` - give the full name including it's version (assumes we include version on the template pdf - that I think we should)

Reply:
Yes, thats a good idea. Yes, we can set the field value to the filename of the template pdf we used as the template. Nice. I'll change the field name 👍

Reply:
I've called it `certificate_template_filename` and set it's width to 255